apps/silverback-gatsby/cypress/integration/constants.ts

```diff
@@ -3,5 +3,5 @@ export const drupalNodeOpUrl = 'http://localhost:8888/node-op';
 export const previewUrl = 'http://localhost:8000';
 export const siteUrl = 'http://localhost:9000';
 
-export const refreshDelay = 10_000;
-export const rebuildDelay = 20_000;
+export const refreshDelay = 15_000;
+export const rebuildDelay = 30_000;
```

Comment:
@pmelab so Gatsby 3 is slower? 😱 

Or was it a wrong assumption and we can revert the delays to smaller values?

Reply:
No, that's what turned the tests green. Maybe we have to rethink the approach there.